### ISSUE TITLE
Support of Lambda web adapter

### ISSUE THREAD

**Issue by billmetangmo** [aws/copilot-cli#5274]
Hello,

Very good job on AWS Copilot.
I am wondering if the possibility to use Copilot containers using Lambda web adapter is in th roadmap ?
This could be a really good feature for dev environment for example

Regards,
BM

**Comment by bvtujo** [comment]
Can you elaborate a little more on what web adapter does and how you might use it with Copilot? It seems like it's a good way of integrating ALBs with lambda, but I wanted to make sure I'm understanding right. 

Would this be a replacement for the underlying compute that Copilot provisions?

**Comment by billmetangmo** [comment]
Yes that's exactly what I mean. Be able to deploy containers in addition to aws apprunner, ecs fargate on lambda as with lambda web adapter a container can be deployed with lambda without code modification.

This can be very useful to deploy worker service or to deploy all services in dev environment to benefit from scale to 0 and reduce pricing.

Do you think it's possible ? I am very excited by this feature and can possibly make a contribution.

**Comment by bvtujo** [comment]
It would need careful investigation, and many of our built-in features rely on parts of the ECS task definition. I think you could connect a lambda web adaptor app to services in the Copilot cluster by making sure that the environment variables are populated correctly, the function is deployed in a VPC, and the security groups are set up right, but I am not sure if we can support serverless transforms or SAM deploy inside copilot at the moment.

This is a really cool feature request.
